tests/aws/Makefile

```diff
@@ -23,16 +23,20 @@ ${PROG}:${SRCS:.c=.o}
 
 main.o: main.c ../../util/warnp.h ../../aws/aws_readkeys.h ../../aws/aws_sign.h
 	${CC} ${CFLAGS_POSIX} -D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700 -DCPUSUPPORT_CONFIG_FILE=\"cpusupport-config.h\"  -I../.. ${IDIRS} ${CPPFLAGS} ${CFLAGS} -c main.c -o main.o
-sha256.o: ../../alg/sha256.c ../../cpusupport/cpusupport.h ../../cpusupport-config.h ../../util/insecure_memzero.h ../../alg/sha256_shani.h ../../alg/sha256_sse2.h ../../util/sysendian.h ../../util/warnp.h ../../alg/sha256.h
+sha256.o: ../../alg/sha256.c ../../cpusupport/cpusupport.h ../../cpusupport-config.h ../../util/insecure_memzero.h ../../alg/sha256_arm.h ../../alg/sha256_shani.h ../../alg/sha256_sse2.h ../../util/sysendian.h ../../util/warnp.h ../../alg/sha256.h
 	${CC} ${CFLAGS_POSIX} -D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700 -DCPUSUPPORT_CONFIG_FILE=\"cpusupport-config.h\"  -I../.. ${IDIRS} ${CPPFLAGS} ${CFLAGS} -c ../../alg/sha256.c -o sha256.o
+sha256_arm.o: ../../alg/sha256_arm.c ../../cpusupport/cpusupport.h ../../cpusupport-config.h ../../util/sysendian.h ../../alg/sha256_arm.h
+	${CC} ${CFLAGS_POSIX} -D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700 -DCPUSUPPORT_CONFIG_FILE=\"cpusupport-config.h\" ${CFLAGS_ARM_SHA256} -I../.. ${IDIRS} ${CPPFLAGS} ${CFLAGS} -c ../../alg/sha256_arm.c -o sha256_arm.o
 sha256_shani.o: ../../alg/sha256_shani.c ../../cpusupport/cpusupport.h ../../cpusupport-config.h ../../alg/sha256_shani.h
 	${CC} ${CFLAGS_POSIX} -D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700 -DCPUSUPPORT_CONFIG_FILE=\"cpusupport-config.h\" ${CFLAGS_X86_SHANI} ${CFLAGS_X86_SSSE3} -I../.. ${IDIRS} ${CPPFLAGS} ${CFLAGS} -c ../../alg/sha256_shani.c -o sha256_shani.o
-sha256_sse2.o: ../../alg/sha256_sse2.c ../../cpusupport/cpusupport.h ../../cpusupport-config.h ../../util/sysendian.h ../../alg/sha256_sse2.h
```

Comment:
It's surprising to see `sse256_sse2.o` being changed in this PR, but this line is just due to 01d99a06489119d15d0f95428f6ce342a71dac0e, and we haven't run `make Makefiles` since that commit.